### ISSUE TITLE
freeablo[2647] <Warning>: displaySetMode: unable to find mode dict in array (Mac)

### ISSUE THREAD

**Issue by gbudny** [wheybags/freeablo#216]
Hi

I see this warning when I try to run Freeablo in the fullscreen mode on Mac:

/Users/gbudny/Desktop/Diablo/build/launcher 
user-Mac-mini:~ gbudny$ [Info] - Loaded font face FreeabloWhite11 (from resources/fonts/Freeablo/FreeabloWhite11.fnt).
[Info] - Loaded font face FreeabloGold16 (from resources/fonts/Freeablo/FreeabloGold16.fnt).
[Info] - Loaded font face FreeabloGold22 (from resources/fonts/Freeablo/FreeabloGold22.fnt).
[Info] - Loaded font face FreeabloGold24 (from resources/fonts/Freeablo/FreeabloGold24.fnt).
[Info] - Loaded font face FreeabloGold30 (from resources/fonts/Freeablo/FreeabloGold30.fnt).
[Info] - Loaded font face FreeabloGold42 (from resources/fonts/Freeablo/FreeabloGold42.fnt).
[Info] - Loaded font face FreeabloSilver16 (from resources/fonts/Freeablo/FreeabloSilver16.fnt).
[Info] - Loaded font face FreeabloSilver24 (from resources/fonts/Freeablo/FreeabloSilver24.fnt).
[Info] - Loaded font face FreeabloSilver30 (from resources/fonts/Freeablo/FreeabloSilver30.fnt).
[Info] - Loaded font face FreeabloSilver42 (from resources/fonts/Freeablo/FreeabloSilver42.fnt).
[Info] - Loaded font face FreeMono BoldOblique (from resources/fonts/FreeMono/FreeMonoBoldOblique.ttf).
[Info] - Loaded font face FreeMono Bold (from resources/fonts/FreeMono/FreeMonoBold.ttf).
[Info] - Loaded font face FreeMono Oblique (from resources/fonts/FreeMono/FreeMonoOblique.ttf).
[Info] - Loaded font face FreeMono Medium (from resources/fonts/FreeMono/FreeMono.ttf).
Diablo.exe v109 detected
w: 25, h: 25
w: 25, h: 23
w: 23, h: 25
w: 23, h: 23
FPS: 62.0079
FPS: 53.3597
0 -- size of effects
0 -- size of effects
w: 25, h: 25
w: 25, h: 23
w: 23, h: 25
w: 23, h: 23
FPS: 60.0394
550FPS: 32.6296
FPS: 62.5
FPS: 60.0394
FPS: 60.0985
FPS: 60.0394
FPS: 60.0394
FPS: 60
tick time exceeded by 40ms
390FPS: 48.2283
FPS: 60.1578
Mar 27 20:13:49 user-Mac-mini.local freeablo[2647] <Warning>: displaySetMode: unable to find mode dict in array

uname -a
Darwin user-Mac-mini.local 14.5.0 Darwin Kernel Version 14.5.0: Tue Sep  1 21:23:09 PDT 2015; root:xnu-2782.50.1~1/RELEASE_X86_64 x86_64

**Comment by wheybags** [comment]
This is yosemite right?
I'll have to get myself a yosemite vm and test this because google isn't turning up anything for this or your other issue :(

**Comment by gbudny** [comment]
Yes. The exact version of this operating system is OS X Yosemite: 10.10.5:

uname -a
Darwin user-Mac-mini.local 14.5.0 Darwin Kernel Version 14.5.0: Tue Sep  1 21:23:09 PDT 2015; root:xnu-2782.50.1~1/RELEASE_X86_64 x86_64

**Comment by wheybags** [comment]
Does this still happen?
